src/ft2font.cpp

```diff
@@ -162,9 +162,19 @@ FT2Image::draw_rect_filled(unsigned long x0, unsigned long y0, unsigned long x1,
 
 inline double conv(long v)
 {
-    return double(v) / 64.0;
+    return v / 64.;
 }
 
+static FT_UInt ft_get_char_index_or_warn(FT_Face face, FT_ULong charcode)
+{
+    FT_UInt glyph_index = FT_Get_Char_Index(face, charcode);
+    if (!glyph_index) {
+        PyErr_WarnEx(NULL, "Required glyph missing from current font.", 1);
```

Comment:
Add `charcode` to message? Also, this might raise; I don't know what you'll need to do to handle that here.

Reply:
What happens when you run with `-Werror` though?

Reply:
Good catch, that gets a `SystemError: <built-in method set_text of matplotlib.ft2font.FT2Font object at 0x7fe670554100> returned a result with an error set`.
Fortunately there's already machinery to handle that (throwing py::exception{} which is handled by CALL_CPP_FULL).  However it (empirically) looks like PyErr_WarnFormat always returns 0 even if it sets the error?  So I explicitly checked for PyErr_Occurred instead, which is fine, but perhaps you have a better idea of why checking the return value of PyErr_WarnFormat doesn't work?